### DISCUSSION TITLE
Enable popups as option

### DISCUSSION THREAD

**Discussion by tmrk** [rocksdanister/lively#1483] [Ideas]
Any chance popups could be enabled as an option? I have a login authentication script via MSAL and neither the redirect nor the popup method is working - and I was thinking that the optional enabling of popups could be the easiest way to solve this?

And just on a somewhat related note: is there a way to open web links in a browser window instead of the built-in view?

100% ❤️ this project, thanks so much for making it!

**Comment by rocksdanister** [comment]
Try WebView2:
https://github.com/rocksdanister/lively/wiki/Web-Player#webview2-microsoft-edge

**Reply by tmrk** [reply]
> Try WebView2: https://github.com/rocksdanister/lively/wiki/Web-Player#webview2-microsoft-edge

Thanks, I have tried doing that but how do I download/install the plugin?

**Reply by rocksdanister** [reply]
https://github.com/rocksdanister/lively/releases/download/v2.0.4.0/lively_webview2.zip
